swagger/api.yaml

```diff
@@ -68,6 +68,9 @@ paths:
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise/260055b/api.yaml#/endpoints/v1/enterpriseCustomerCourseEnrollments"
   "/enterprise/v2/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/8fc95e4/api.yaml#/endpoints/v1/enterpriseCustomerLearnerSummary"
+  # Learner Summary endpoint powered by Learner Progress Report V1 API
+  "/enterprise/v3/enterprise-customer/{uuid}/learner-summary":
+    $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/*******/api.yaml#/endpoints/v2/enterpriseCustomerLearnerSummary"
```

Comment:
**Note:** commit hash will be updated once https://github.com/edx/edx-enterprise-data/pull/264 is merged